src/ui/profile_preferences.ui

```diff
@@ -525,30 +525,93 @@
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_spellCheck">
+         <property name="toolTip">
+          <string notr="true"/>
+         </property>
          <property name="title">
-          <string>Spell check</string>
+          <string>Spell checking</string>
          </property>
-         <layout class="QGridLayout" name="gridLayout_groupBox_spellCheck">
-          <item row="0" column="0">
-           <widget class="QLabel" name="label_57">
+         <layout class="QVBoxLayout" name="verticalLayout_2">
+          <item>
+           <widget class="QCheckBox" name="checkBox_spellCheck">
+            <property name="toolTip">
+             <string>&lt;p&gt;This option controls spell-checking on the command line in the main console for this profile.&lt;/p&gt;&lt;p&gt;&lt;i&gt;The dictionary selected below will be available to the lua sub-system whether this is checked or not.&lt;/i&gt;&lt;/p&gt;</string>
```

Comment:
> The dictionary selected below will be available to the lua sub-system whether this is checked or not.

This is a good tidbit of information but I don't think this is the right place for it, documentation for the Lua API would be better.

Reply:
It does help to clarify the action of the control - so why is it *not* a right place for it? :confused: Indeed if the user only wants the dictionary for the lua subsystem (which is a bit of an odd case I admit) then this will reassure them as to its effects.

Reply:
I see your point in being helpful to explain what the control does - but most people aren't coders who'll understand what does "lua sub-system" mean, right?

It's not the right place for it because if you want information on the Lua functionality, you look in the documentation, not the tooltips. It's the same reason we don't talk about the Lua API for the mapper on any of the mapper-related dialogs or menus.

Reply:
Okay - I'll take out the *second* paragraph.